src/main/target/OMNIBUSF4/CL_RACINGF4.mk

```diff
@@ -0,0 +1,11 @@
+#==CL_RACING F4 PDB rev 1.0==
+#Owner: CL_RACING desgin base on OmnibusF4 board
+#Board information:
+#- CPU - STM32F405RGT6
+#- MPU-6000
+#- Full push in SD Card Slot 
+#- Build in 5V BEC 
+#- Build in LC filter for camera and vtx
+#- Build in OSD
+#- Built in current meter, PDB
+FEATURES       += VCP SDCARD
```

Comment:
Can you please add this into `src/main/target/OMNIBUSF4/target.mk`, guarded by `ifeq ($(TARGET), CL_RACINGF4)`? CL_RACINGF4.mk should be an empty file.